src-extras/Database/LSMTree/Extras/RunData.hs

```diff
@@ -163,11 +153,14 @@ unsafeCreateRunAt ::
   -> IO (Ref (Run IO h))
 unsafeCreateRunAt fs hbio indexType fsPaths (RunData m) = do
     let blobpath = FS.addExtension (runBlobPath fsPaths) ".wb"
+    let runParams = RunParams {
+          runParamCaching = CacheRunData,
+          runParamAlloc   = RunAllocFixed 10,
+          runParamIndex   = indexType
+        }
```

Comment:
I was assuming we'd replace the `IndexType` argument to the `unsafeCreateRunAt` function by a `RunParams`

Reply:
The changes you made are what I had in mind